### PR TITLE
KMM: Adding e2e-hub job.

### PR DESCRIPTION
**Pull request by ybettan**
Also adding a timeout for the operators installation.

### CODE REVIEW

**Comment by ybettan** [comment]
/pj-rehearse auto-ack
/hold
Blocked by https://github.com/rh-ecosystem-edge/kernel-module-management/pull/593

**Comment by ybettan** [comment]
Re-pushed to try to trigger the rehearsal jobs.

File: ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml [hunk 1]
```diff
@@ -41,6 +41,14 @@ images:
     RUN rm -rf /etc/yum.repos.d/*
     RUN curl http://base-4-13-rhel9.ocp.svc > /etc/yum.repos.d/base-4-13-rhel9.repo
   to: ubi9-minimal-entitled
+- dockerfile_literal: |
+    FROM src
+    # Installing the `oc` client.
+    RUN curl --retry 3 -SL https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz \
+        | tar -xz -C /usr/local/bin
```
Comment (qbarrand) [inline ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml:48]: We use `cli: latest`, so I don't think this is needed.

File: ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml [hunk 2]
```diff
@@ -41,6 +41,14 @@ images:
...
+    RUN pip3 install yq
+  from: src
+  to: test-bin
```
Comment (qbarrand) [inline ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml:51]: Could we give it a more explicit name? This image is only used in the `e2e-hub` test.

File: ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml [hunk 3]
```diff
@@ -41,6 +41,14 @@ images:
     RUN rm -rf /etc/yum.repos.d/*
     RUN curl http://base-4-13-rhel9.ocp.svc > /etc/yum.repos.d/base-4-13-rhel9.repo
   to: ubi9-minimal-entitled
+- dockerfile_literal: |
+    FROM src
+    # Installing the `oc` client.
+    RUN curl --retry 3 -SL https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz \
+        | tar -xz -C /usr/local/bin
+    RUN pip3 install yq
```
Comment (qbarrand) [inline ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml:49]: Shall we use the [Go `yq`](https://github.com/mikefarah/yq) instead? It has 3 to 4 times the stars on GitHub, and is available as a single binary.

File: ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml (hunk 2 above)
Comment (ybettan) [reply ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml:51]: How about `ocp-toolkit`?

Comment (qbarrand) [reply ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml]: Or `ocp-release-yq`?

File: ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml (hunk 1 above)
Comment (ybettan) [reply ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml]: Good point. Removed.

File: ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml (hunk 2 above)
Comment (ybettan) [reply ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml]: But then if we add some packages to it the the name won't be relevant anymore.

File: ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml (hunk 3 above)
Comment (ybettan) [reply ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml]: Sure. Replaced it with the Go yq.

**Pull request merged**